conf/airframes/ENAC/cyfoam.xml

```diff
@@ -0,0 +1,307 @@
+
+      <!--run the chimera mag once every 64 seconds to avoid message spam-->
+      <define name="MPU9250_MAG_PRESCALER" value="32768"/>
+    </module>
+
+    <module name="send_imu_mag_current">
+      <define name="MILLIAMP_AT_IDLE_THROTTLE" value="1000"/>
+      <define name="MILLIAMP_AT_FULL_THROTTLE" value="30000"/>
+      <define name="CURRENT_ESTIMATION_NONLINEARITY" value="1.5"/>
+    </module>
+    <module name="sys_mon"/>
+
+    <module name="telemetry"     type="xbee_api"/>
+    <module name="imu" type="chimera">
+      <define name="IMU_MPU9250_GYRO_RANGE" value="MPU9250_GYRO_RANGE_2000" />
+      <define name="IMU_MPU9250_ACCEL_RANGE" value="MPU9250_ACCEL_RANGE_16G" />
+    </module>
+
+    <!--<module name="servo_tester"/>-->
+
+    <module name="air_data">
+      <define name="USE_AIRSPEED_AIR_DATA" value="FALSE"/>
+    </module>
+    <module name="gps"           type="ublox">
+      <configure name="GPS_BAUD" value="B115200"/>
+    </module>
+    <module name="stabilization" type="indi"/>
+    <module name="guidance"      type="indi">
+      <define name="GUIDANCE_INDI_RC_DEBUG" value="FALSE"/>
+      <define name="GUIDANCE_INDI_POS_GAIN" value="0.2"/>
+      <define name="GUIDANCE_INDI_SPEED_GAIN" value="1.0"/>
+      <define name="GUIDANCE_INDI_POS_GAINZ" value="0.5"/>
+      <define name="GUIDANCE_INDI_SPEED_GAINZ" value="1.8"/>
+      <define name="GUIDANCE_INDI_PITCH_LIFT_EFF" value="0.12"/>
+      <define name="GUIDANCE_INDI_PITCH_EFF_SCALING" value="1.0"/>
+      <define name="GUIDANCE_H_REF_MAX_SPEED" value="18.0"/> <!--not used-->
+      <define name="GUIDANCE_INDI_MIN_THROTTLE" value="4000"/>
+      <define name="GUIDANCE_INDI_MIN_THROTTLE_FWD" value="1500"/>
+      <define name="GUIDANCE_INDI_MAX_AIRSPEED" value="16.0"/>
+      <define name="GUIDANCE_HEADING_IS_FREE" value="FALSE"/> <!--heading can not be set by navigation-->
+      <!--<define name="KNIFE_EDGE_TEST" value="TRUE"/>-->
+      <!--Flap effectiveness on lift-->
+      <define name="FE_LIFT_A_PITCH" value="0.00018"/>
+      <define name="FE_LIFT_B_PITCH" value="0.00072"/>
+      <define name="FE_LIFT_A_AS" value="0.0008"/>
+      <define name="FE_LIFT_B_AS" value="0.00009"/>
+    </module>
+
+    <!--<module name="ahrs"          type="int_cmpl_quat">-->
+      <!--<configure name="USE_MAGNETOMETER" value="TRUE"/>-->
+      <!--[>Use external magnetometer<]-->
+      <!--<define name="AHRS_ICQ_MAG_ID" value="MAG_HMC58XX_SENDER_ID"/>-->
+      <!--[><define name="AHRS_GRAVITY_UPDATE_COORDINATED_TURN" value="TRUE"/><]-->
+      <!--[><define name="AHRS_GPS_SPEED_IN_NEGATIVE_Z_DIRECTION" value="TRUE"/><]-->
+      <!--</module>-->
+
+    <!--<module name="ins"/>-->
+    <module name="ins" type="float_invariant">
+      <define name="INS_PROPAGATE_FREQUENCY" value="500"/>
+      <define name="INS_FINV_MAG_ID" value="MAG_HMC58XX_SENDER_ID"/>
+    </module>
+
+  </firmware>
+
+  <modules main_freq="512">
+    <!--<module name="gps" type="ubx_ucenter"/>-->
```

Comment:
We are phasing out separate modules sections, can you move this to the firmware above (like you already did with your other modules?) additionally, I think the order the modules appear is important for how they are run so consider that, usually, the "subsystems" go first then the more mission specific modules.